styleguide.config.js

```diff
@@ -26,6 +26,8 @@ module.exports = {
             components: "packages/wonder-blocks-core/components/*.js",
         },
         {
+            // NOTE(jeresig): Work-in-progress.
+            private: true,
```

Comment:
Does it make sense to make this manual? Should we change the script behavior to exclude sections that map to a private/prototype repo when we're producing the prod docs? Seems like that would be the better approach, with less effort and less margin for error when something moves out of private/prototype.

Reply:
OK, my changes to the script are in master now, so this manual setting of WIP sections is no longer needed! 🎊

Reply:
Removed this!